src/elm/Cambiatus/Object/Community.elm

```diff
@@ -64,6 +64,11 @@ hasShop =
     Object.selectionForField "Bool" "hasShop" [] Decode.bool
 
 
+hasKyc : SelectionSet Bool Cambiatus.Object.Community
+hasKyc =
+    Object.selectionForField "Bool" "hasKyc" [] Decode.bool
```

Comment:
@victorgdev I guess you added this `hasKyc` by hand. I'm sorry, but this works a bit different :)

Everything from `src/elm/Cambiatus` folder is generated automatically by running `yarn generate-graphql` from the command line. This command queries the backend and creates those Elm modules with types according to the data, received from the backend. We use [elm-graphql](https://github.com/dillonkearns/elm-graphql) package for it.

We should not change these files by hand, they should be changed only with `yarn generate-graphql`.

I know, that's quite unintuitive and it looks like every new developer goes through this issue, but that's how it works 🤷🏻‍♂️

So, please, run `yarn generate-graphql` from the root of the project and commit/push all those files you'll get created. There will be several new files and several files with modifications because we added some new features to the backend but now used them on the fronted yet.

Reply:
Ok!! I will do it!